addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -18,15 +18,23 @@
         <li t-elif="aside_slide.channel_id.enroll == 'payment'" class="text-decoration-none small">
             <i class="fa fa-download me-1"/>
             <t t-call="website_sale_slides.course_buy_course_link">
+                <t t-set="for_resources" t-value="1"/>
                 <t t-set="slide" t-value="aside_slide"/>
             </t>
         </li>
     </xpath>
 </template>
 
-<template name="Buy Course To Download Resource Slide Detail" id="slide_content_detailed_buy_course" inherit_id="website_slides.slide_content_detailed">
+<template name="Buy Course To Access Resources or Interact Slide Detail" id="slide_content_detailed_buy_course" inherit_id="website_slides.slide_content_detailed">
     <xpath expr="//div[hasclass('o_wslides_js_course_join') and hasclass('o_wslides_no_access')]" position="inside">
         <span t-elif="slide.channel_id.enroll == 'payment'" class="text-muted me-auto border-start ps-3">
+            <t t-call="website_sale_slides.course_buy_course_link">
+                <t t-set="for_resources" t-value="1"/>
+            </t>
+        </span>
+    </xpath>
+    <xpath expr="//span[hasclass('o_wslides_js_course_join') and hasclass('o_wslides_no_access')]" position="inside">
```

Comment:
Should probably make that span a div instead and rely on something else to make a behavior difference ("for_resources" switched on/off).

From what I understand this span is for the "comments" part?
So it could be `//div[hasclass('comments')]//div[hasclass('o_wslides_js_course_join')]` to correctly target what we want maybe?

Relying on tag type for behaviors is not super robust.